### PR TITLE
Fix user-based UI element visibility issues

### PR DESCRIPTION
**Pull request by ian-ross**
Fix 183: project page ellipsis menu visibility
Fix 184: add project organisation filtering
Fix 185: display of project add button

(Some of the code added here will be cleaned up and rationalised as part of #485.)

### CODE REVIEW

File: cadasta/organization/tests/test_views_default_projects.py [hunk 1]
```diff
@@ -472,6 +489,7 @@ def test_full_flow_valid(self):
             reverse('project:add'), self.DETAILS_POST_DATA
         )
         assert details_response.status_code == 200
+        print(details_response)
```
Comment (oliverroick) [inline cadasta/organization/tests/test_views_default_projects.py]: We should remove `print` statements.

**Comment by oliverroick** [comment]
- #185 is not sufficiently addressed. The "Add project" button is still visible from the organization dashboard.
- #183 looks ok. Would it be possible to add the same functionality to the organization dashboard page? I.e. when I don't have edit/archive permissions, the ellipsis menu is not displayed.
- #184 is ok.

File: cadasta/organization/tests/test_views_default_projects.py (hunk 1 above)
Comment (ian-ross) [inline cadasta/organization/tests/test_views_default_projects.py]: We should, but sometimes we forget...

**Comment by ian-ross** [comment]
All fair comments.  I'll fix these things.  I'd not thought about doing the organization dashboard as well, but it's definitely in scope for this.

**Comment by ian-ross** [comment]
Some of this isn't very pretty.  I've been experimenting with a couple of different approaches to doing these "pre-request" permissions checks.  I'm planning to consolidate and clean this stuff up when I do #485.

**Pull request merged**